v20.1/cockroachcloud-upgrade-to-v20.1.md

```diff
@@ -0,0 +1,163 @@
+
+Before your upgrade has been finalized, remember to prevent new schema changes and changes to user privileges as mention [earlier](#review-temporary-limitations).
+
+Also, most v20.1 features can be used right way, but there are some that will be enabled only after the upgrade has been finalized. Attempting to use these features before then will result in errors:
+
+- **Primary key changes:** After finalization, it will be possible to change the primary key of an existing table using the [`ALTER TABLE ... ALTER PRIMARY KEY`](alter-primary-key.html) statement, or using [`DROP CONSTRAINT` and then `ADD CONSTRAINT`](https://www.cockroachlabs.com/docs/v20.1/add-constraint.html#drop-and-add-a-primary-key-constraint) in the same transaction.
+
+- **Dropping indexes used by foreign keys:** After finalization, it will be possible to drop an index used by a foreign key constraint if another index exists that fulfills the [indexing requirements](https://www.cockroachlabs.com/docs/v20.1/foreign-key.html#rules-for-creating-foreign-keys).
+
+- **Hash-sharded indexes:** After finalization, it will be possible to use [hash-sharded indexes](https://www.cockroachlabs.com/docs/v20.1/create-index.html#create-a-hash-sharded-secondary-index) to distribute sequential traffic uniformly across ranges, eliminating single-range hotspots and improving write performance on sequentially-keyed indexes. This is an experimental feature that must be enabled by setting the `experimental_enable_hash_sharded_indexes` session variable to `on`.
+
+- **`CREATEROLE` and `NOCREATEROLE` privileges:** After finalization, it will be possible to [allow or disallow a user or role to create, alter, or drop other roles](https://www.cockroachlabs.com/docs/v20.1/create-user.html#allow-the-user-to-create-other-users) via the `CREATEROLE` or `NOCREATEROLE` privilege.
+
+- **Nested transactions:** After finalization, it will be possible to create [nested transactions](https://www.cockroachlabs.com/docs/v20.1/transactions.html#nested-transactions) using [`SAVEPOINT`s](savepoint.html).
+
+- **`TIMETZ` data type:** After finalization, it will be possible to use the [`TIMETZ`](https://www.cockroachlabs.com/docs/v20.1/time.html#timetz) data type to store a time of day with a time zone offset from UTC.
+
+- **`TIME`/`TIMETZ` and `INTERVAL` precision:** After finalization, it will be possible to specify precision levels from 0 (seconds) to 6 (microseconds) for [`TIME`/`TIMETZ`](https://www.cockroachlabs.com/docs/v20.1/time.html#precision) and [`INTERVAL`](https://www.cockroachlabs.com/docs/v20.1/interval.html#precision) values.
+
+## Step 6. Finish the upgrade
+
+During the 24-hour window before the upgrade is automatically finalized, if you see unexpected behavior, you can trigger a rollback to v19.2. If everything looks good, you also have the choice to finalize the upgrade more quickly so as to lift the [temporary limitations](#review-temporary-limitations) in place during the upgrade.
```

Comment:
Add link to rollback instructions

Reply:
I don't want to overload this paragraph with links. The rollback and finalize instructions are directly below (part of this step). The link to temporary limitations is earlier, and I don't want that link to get lost.